webapp/platform/types/src/integrations.ts

```diff
@@ -134,6 +134,7 @@ export type IntegrationsState = {
     appsBotIDs: string[];
     systemCommands: IDMappedObjects<Command>;
     commands: IDMappedObjects<Command>;
+    dialogArguments?: CommandArgs;
```

Comment:
I'm not sure if we should be reusing this type here because `CommandArgs` has an extra field and may later be changed in a way that doesn't fit for interactive dialogs